packages/ethereum/ethereum-light-client/src/consensus_state.rs

```diff
@@ -13,6 +13,8 @@ use crate::{error::EthereumIBCError, header::ActiveSyncCommittee, verify::BlsVer
 pub struct ConsensusState {
     /// The slot number of the finalized header
     pub slot: u64,
+    /// The execution block number
+    pub execution_block_number: u64,
```

Comment:
Even if we were to do this, wouldn't it better for this to be stored in client state. `latest_execution_block_number`?

Reply:
Not sure why that would be better?